packages/headless/docs.config.json

```diff
@@ -0,0 +1,279 @@
+      "related": []
+    },
+    {
+      "name": "Sort",
+      "initialize": {
+        "initializer": {
+          "name": "buildSort",
+          "source": "src/controllers/sort/headless-sort.ts"
+        },
+        "others": [
+          {
+            "name": "SortProps",
+            "source": "src/controllers/sort/headless-sort.ts"
+          },
+          {
+            "name": "SortInitialState",
+            "source": "src/controllers/sort/headless-sort.ts"
+          }
+        ]
+      },
+      "state": "SortState",
+      "related": [
+        {
+          "name": "SortOrder",
+          "source": "src/features/sort-criteria/criteria.ts"
+        },
+        {
+          "name": "SortBy",
+          "source": "src/features/sort-criteria/criteria.ts"
+        },
+        {
+          "name": "SortByRelevancy",
+          "source": "src/features/sort-criteria/criteria.ts"
+        },
+        {
+          "name": "SortByQRE",
+          "source": "src/features/sort-criteria/criteria.ts"
+        },
+        {
+          "name": "SortByDate",
+          "source": "src/features/sort-criteria/criteria.ts"
+        },
+        {
+          "name": "SortByField",
+          "source": "src/features/sort-criteria/criteria.ts"
+        },
+        {
+          "name": "SortByNoSort",
+          "source": "src/features/sort-criteria/criteria.ts"
+        },
+        {
+          "name": "SortCriterion",
+          "source": "src/features/sort-criteria/criteria.ts"
+        },
+        {
+          "name": "buildCriterionExpression",
+          "source": "src/features/sort-criteria/criteria.ts"
+        },
+        {
+          "name": "buildRelevanceSortCriterion",
+          "source": "src/features/sort-criteria/criteria.ts"
+        },
+        {
+          "name": "buildDateSortCriterion",
+          "source": "src/features/sort-criteria/criteria.ts"
+        },
+        {
+          "name": "buildFieldSortCriterion",
+          "source": "src/features/sort-criteria/criteria.ts"
+        },
+        {
+          "name": "buildQueryRankingExpressionSortCriterion",
+          "source": "src/features/sort-criteria/criteria.ts"
+        },
+        {
+          "name": "buildNoSortCriterion",
+          "source": "src/features/sort-criteria/criteria.ts"
+        }
+      ]
+    },
+    {
+      "name": "Tab",
+      "initialize": {
+        "initializer": {
+          "name": "buildTab",
+          "source": "src/controllers/tab/headless-tab.ts"
+        },
+        "others": [
+          {
+            "name": "TabProps",
+            "source": "src/controllers/tab/headless-tab.ts"
+          },
+          {
+            "name": "TabOptions",
+            "source": "src/controllers/tab/headless-tab.ts"
+          },
+          {
+            "name": "TabInitialState",
+            "source": "src/controllers/tab/headless-tab.ts"
+          }
+        ]
+      },
+      "state": "TabState",
+      "related": []
+    }
+  ]
+}
```

Comment:
I think it could be helpful to make the config a `.ts` file at some point to get the benefits of type-checking on the structure of the objects. We can look at this as part of a future interation though,